src/main/java/uk/gov/mca/beacons/api/dto/ErrorResponseDTO.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.mca.beacons.api.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+
+@Getter
+public class ErrorResponseDTO {
+
+  private final List<ValidationError> errors = new ArrayList<>();
```

Comment:
See [JSON API spec](https://jsonapi.org/format/#errors).  I haven't included the other option keys in the response object as I didn't see a need to put them in for now but we can always revisit it later if we need to add these